spec/CachePluginSpec.php

```diff
@@ -41,18 +41,25 @@ function it_caches_responses(CacheItemPoolInterface $pool, CacheItemInterface $i
         $response->getBody()->willReturn($stream);
         $response->getHeader('Cache-Control')->willReturn(array());
         $response->getHeader('Expires')->willReturn(array());
+        $response->getHeader('ETag')->willReturn(array());
 
         $pool->getItem('e3b717d5883a45ef9493d009741f7c64')->shouldBeCalled()->willReturn($item);
         $item->isHit()->willReturn(false);
-        $item->set(['response' => $response, 'body' => $httpBody])->willReturn($item)->shouldBeCalled();
-        $item->expiresAfter(60)->willReturn($item)->shouldBeCalled();
+        $item->set()->willReturn($item)->shouldBeCalled();
```

Comment:
I can't figure out a good way to write this line. I cant specify the exact data since it is dependent on `time()`. 

Can I say `$item->set(<any argument>)->willRetu....`?